plugins/rest_api_plugin.py

```diff
@@ -296,7 +303,8 @@ def get_config_boolean_value(section, key, default_value):
             {"name": "ignore_first_depends_on_past", "description": "Ignores depends_on_past dependencies for the first set of tasks only (subsequent executions in the backfill DO respect depends_on_past).", "form_input_type": "checkbox", "required": False},
             {"name": "subdir", "description": "File location or directory from which to look for the dag", "form_input_type": "text", "required": False},
             {"name": "pool", "description": "Resource pool to use", "form_input_type": "text", "required": False},
-            {"name": "dry_run", "description": "Perform a dry run", "form_input_type": "checkbox", "required": False}
+            {"name": "dry_run", "description": "Perform a dry run", "form_input_type": "checkbox", "required": False},
+            {"name": "reset_dagruns", "description": "if set, the backfill will delete existing backfill-related DAG runs and start anew with fresh, running DAG runs", "form_input_type": "checkbox", "required": False}
```

Comment:
Were we able to get this to work? I recall you saying you had issues with it prompting you to confirm yes or no

Reply:
This is not working if there is any failed instances. So, I have reverted this change.